src/peoplemeasurement/csv_imports/sensor_csv_importer.py

```diff
@@ -13,18 +13,18 @@ class SensorCsvImporter(CsvImporter):
     def _import_csv_reader(self, csv_reader) -> int:
         with transaction.atomic():
             if Sensors.objects.count() > 0:
-                self._truncate_sensors()
+                self._truncate()
 
-            sensors = []
+            obj_dicts = []
             for row in csv_reader:
-                sensors.append(self._create_sensor_for_row(row))
+                obj_dicts.append(self._create_obj_dict_for_row(row))
 
-            if sensors:
-                Sensors.objects.bulk_create(sensors)
+            if obj_dicts:
+                Sensors.objects.bulk_create(obj_dicts)
 
-        return len(sensors)
+        return len(obj_dicts)
```

Comment:
Deze code herhalen we vier keer, dus kan abstracter in de `CsvImporter` base class:

```

    def _import_csv_reader(self, model: models.Model, csv_reader) -> int:
        with transaction.atomic():
            if model.objects.count() > 0:
                self._truncate()

            obj_dicts = []
            for row in csv_reader:
                obj_dicts.append(self._create_obj_dict_for_row(row))

            if obj_dicts:
                model.objects.bulk_create(obj_dicts)

        return len(obj_dicts)
```

Vervolgens hoef je in de implementaties enkel het volgende te doen:

```
def _import_csv_reader(self, csv_reader) -> int:
    super()._import_csv_reader(self, Sensor, csv_reader)
```